README.md

```diff
@@ -175,10 +175,11 @@ bpf-next to Github sync
 All the gory details of syncing can be found in `scripts/sync-kernel.sh`
 script.
 
-Some header files in this repo (`include/linux/*.h`) are reduced versions of
```

Comment:
hold on, here we are talking about kernel-internal include/linux/*.h headers, not UAPI headers

Reply:
Aha, I guess I misunderstood this paragraph. Maybe another paragraph about include/uapi/linux/*.h?

Reply:
Sigh, I keep missing comments on GitHub. :(